modules/virt-reclaiming-statically-provisioned-persistent-volumes.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reusing-persistent-volumes.adoc
+
+[id="virt-reclaiming-statically-provisioned-persistent-volumes_{context}"]
+= Reclaiming statically provisioned persistent volumes 
+
+Reclaiming a statically provisioned persistent volume (PV) for re-use is dependent on the underlying storage.
+This procedure provides a general approach that might need to be customized depending on your storage.
+
+.Procedure
+
+. Ensure that the reclaim policy of the PV is set to `Retain`:
++
+----
+$ oc get pv <pv_name> -o yaml | grep 'persistentVolumeReclaimPolicy'
+----
+
+.. If the `persistentVolumeReclaimPolicy` is not set to `Retain`, edit the reclaim policy with the following command:
```

Comment:
This nesting feels weird. I'd either unnest this step or add an "intro" step and make both this step and the previous steps nested into the same substeps, like

> . Ensure that the reclaim policy of the PV is set to `Retain`
.. Check the reclaim policy of the PV:
.. If the `persistentVolumeReclaimPolicy`...